### ISSUE TITLE
r-builder causes problem

### ISSUE THREAD

**Issue by sckott** [jimhester/covr#107]
Curious if you have seen this problem before. I use Codecov and normally works great. Trying out @gaborcsardi `r-builder` and now sending the payload to Codecov doesn't work. 

E.g. see https://travis-ci.org/ropensci/binomen/jobs/77516586#L3110 where I get `"token or build not found"`

Codecov suggests we don't need to worry about tokens for Travis builds. I wonder if it could be that.

Or perhaps `covr` isn't able to do it's thing correctly given `r-builder` diff. from `r-travis`, starts here https://travis-ci.org/ropensci/binomen/jobs/77516586#L3045 for this build

**Comment by gaborcsardi** [comment]
I forgot what the problem was, but here is a setup that works:https://github.com/gaborcsardi/argufy/blob/master/.travis.yml

**Comment by sckott** [comment]
@gaborcsardi thanks!

**Comment by sckott** [comment]
@gaborcsardi hmmm, i'm not getting the payload on successful build, will keep exploring

**Comment by jimhester** [comment]
I think the issue is when using a matrix build (like r-builder does) codecov doesn't find the travis job properly.

In the argufy example I had to also specify CODECOV_TOKEN as a protected environment variable (which then precludes coverage for pull requests unfortunately).

I think this is simply a bug / limitation in codecov right now.  I opened https://github.com/codecov/support/issues/78 to see about getting this use-case supported.
